x-pack/plugins/cases/public/components/case_view/components/case_view_alerts.tsx

```diff
@@ -40,8 +41,12 @@ export const CaseViewAlerts = ({ caseData }: CaseViewAlertsProps) => {
     alertsTableConfigurationRegistry: triggersActionsUi.alertsTableConfigurationRegistry,
     configurationId: caseData.owner,
     id: `case-details-alerts-${caseData.owner}`,
+    flyoutState: alertFeatureIds.includes('siem')
```

Comment:
I think this is an implementation detail and should be kept internal to the alerts table. If something changes in the future it will change in one place. The table is aware of the `featureIds`. What do you think?

Reply:
+1. Cases should not be responsible of passing this. The triggers actions ui plugin has all the information required to figure it

Reply:
Yes you are right, it does not have to be like that

Reply:
I will change that in future PR